requirements.txt

```diff
@@ -0,0 +1,15 @@
+fastapi==0.110.1
```

Comment:
What commands are you running locally to install your requirements? Are you doing this via a local virtual environment? Docker container?

Reply:
I will containerize the application but later will create a separate pull request. For now I am running in a local virtual environment.

Reply:
It seems you are using poetry as well (via the `pyproject.toml` file present in your PR) -- it might be useful to just use poetry to manage your dependencies as well here

Reply:
Initially, I was using pyproject.toml for pycharm config but now I have moved all the dependencies to the poetry.